compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -128,6 +128,15 @@ if (cg->profiledPointersRequireRelocation() && secondChild->getOpCodeValue() ==
    }
 #endif
 
+   int64_t secondChildValue = is64bit ? secondChild->getLongInt() : secondChild->getInt();
+   if ((TR::CC_EQ == cc || TR::CC_NE == cc)
+         && 0 == secondChildValue)
```

Comment:
This style, while functionally correct, is different than the style used in the rest of the compiler code base.  Please reverse the operands of the operators.